### PR TITLE
[feature] Improve storage: Update DBStorage and FileStorage, adding t…

### PR DESCRIPTION
**Pull request by Leidysalda**
…wo  new methods.

modified:   models/engine/db_storage.py
modified:   models/engine/file_storage.py
modified:   tests/test_models/test_engine/test_db_storage.py
modified:   tests/test_models/test_engine/test_file_storage.py

### CODE REVIEW

**Comment by Leidysalda** [comment]
Juan, I have reviewed your code  and

File: models/engine/db_storage.py [hunk 1]
```diff
@@ -74,3 +74,16 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
```
Comment (Leidysalda) [inline models/engine/db_storage.py:75]: Juan, I have reviewed the your code and noted that you use unnecessary variables for example, you could have avoided use item variable

File: tests/test_models/test_engine/test_file_storage.py [hunk 2]
```diff
@@ -70,9 +70,16 @@ def test_fs_func_docstrings(self):
 
 class TestFileStorage(unittest.TestCase):
     """Test the FileStorage class"""
+
+    @classmethod
+    def setUpClass(cls):
+        """Set up for the DBStorage tests"""
+        # cls.state = State()
+        # cls.State.save()
+
```
Comment (Leidysalda) [inline tests/test_models/test_engine/test_file_storage.py:79]: Juan, I noted a  issues in comments at use hashtag (#) and quotations (""") in the comments and avoid write code that is not used.

File: models/engine/db_storage.py [hunk 3]
```diff
@@ -74,3 +74,16 @@ def reload(self):
...
         self.__session.remove()
+
+    def get(self, cls, id):
+        """ returns the object based on the class name and its ID"""
+        if cls and id:
+            objs = self.all(cls)
+            item = "{}.{}".format(cls.__name__, id)
```
Comment (Leidysalda) [inline models/engine/db_storage.py:82]: Juan, I have reviewed the your code and noted that you use unnecessary variables for example, you could have avoided use item variable

File: models/engine/db_storage.py [hunk 4]
```diff
@@ -74,3 +74,16 @@ def reload(self):
...
+            return objs[item]
```
Comment (Leidysalda) [inline models/engine/db_storage.py:83]: Juan, I have reviewed the your code and noted that you use unnecessary variables for example, you could have avoided use 'item' variable

File: tests/test_models/test_engine/test_db_storage.py [hunk 5]
```diff
@@ -86,3 +93,24 @@ def test_new(self):
     @unittest.skipIf(models.storage_t != 'db', "not testing db storage")
     def test_save(self):
         """Test that save properly saves objects to file.json"""
+
+    def test_count(self):
+        """ Test for the exist object """
+        # storage = DBStorage()
+        storage = models.storage
+
```
Comment (Leidysalda) [inline tests/test_models/test_engine/test_db_storage.py:101]: Juan, the comments must improve about style issues.

File: tests/test_models/test_engine/test_file_storage.py (hunk 2 above)
Comment (juan-skill) [reply tests/test_models/test_engine/test_file_storage.py:79]: I will take your opinion about to use quotes in python. It's a good habit

File: models/engine/db_storage.py (hunk 4 above)
Comment (juan-skill) [reply models/engine/db_storage.py:83]: ok, I understand you. The next episodios I will try to do it

File: tests/test_models/test_engine/test_db_storage.py (hunk 5 above)
Comment (juan-skill) [reply tests/test_models/test_engine/test_db_storage.py:101]: Again, I will change this habit

**Pull request merged**